.github/workflows/events-ocasional.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
           DRY_RUN: "false"
           PR_FILTER: "labelled"
-          PR_LABELS: "0:] Ready-To-Go,has conflicts"
+          PR_LABELS: "0:] Ready-To-Go"
```

Comment:
is this a smiley face btw? :smile_cat:

Reply:
silent :D or they ll drop it :D
my hidden play :P
and also bc of 0 it is on top of the label list...

Reply:
ok very funny xD